library.c

```diff
@@ -2280,6 +2302,29 @@ PHP_REDIS_API void redis_free_socket(RedisSock *redis_sock)
     efree(redis_sock);
 }
 
+#define PHP_REDIS_COMPRESSION_RATIO_CHECK(packed, unpacked) \
```

Comment:
I'm still not sure about compression minimums but I'm ok if you think that this is fine

Reply:
How about I move it to a different PR and we can bikeshed it there :joy: